.github/workflows/ci.yml

```diff
@@ -196,7 +196,10 @@ jobs:
         sanitizer: [address, leak, memory, thread]
     steps:
     - uses: actions/checkout@v4
-    - uses: dtolnay/rust-toolchain@nightly
+    # FIXME: revert to using the latest nightly when https://github.com/rust-lang/rust/issues/146465 is fixed.
+    - uses: dtolnay/rust-toolchain@master
+      with:
+        toolchain: nightly-2025-09-05
```

Comment:
Please revert this, we'll just ignore the failure. I don't want to use an outdated nightly version that we'll never update.